doc/devbox_setup.md

```diff
@@ -71,7 +71,7 @@ cd cmake
   cmake .. -G "Visual Studio 16 2019" -A Win32
 ```
 
-> This builds x86 libraries. To build for x64 for Visual Studio 2017, `cmake .. -G "Visual Studio 15 2017 Win64"` or for Visual Studio 2019, `cmake .. -G "Visual Studio 16 2019 -A x64"`
+> This builds x86 libraries. To build for x64 for Visual Studio 2017, `cmake .. -G "Visual Studio 15 2017 Win64"` or for Visual Studio 2019, `cmake .. -G "Visual Studio 16 2019" -A x64`
```

Comment:
Please add VS 2022 as well. Here's the generator list from cmake: 

Verified that `cmake .. -G "Visual Studio 17 2022" -A x64` works on my devbox.

```
The following generators are available on this platform (* marks default):
* Visual Studio 17 2022        = Generates Visual Studio 2022 project files.
                                 Use -A option to specify architecture.
  Visual Studio 16 2019        = Generates Visual Studio 2019 project files.
                                 Use -A option to specify architecture.
  Visual Studio 15 2017 [arch] = Generates Visual Studio 2017 project files.
                                 Optional [arch] can be "Win64" or "ARM".
```

Reply:
Please note, our 1ES images still build on 2017.

Reply:
> Please note, our 1ES images still build on 2017.

Is there anything I should specify about that in the doc?